### PR TITLE
Alignment summary

### PR DESCRIPTION
**Pull request by yunfangjuan**
coverage report and etc

### CODE REVIEW

**Comment by yunfangjuan** [comment]
WIP

**Comment by boris-dimitrov** [comment]
or i'd say lgtm so far, hoping pylint passes ;)

File: idseq_pipeline/commands/accessionid2seq_functions.py [hunk 1]
```diff
@@ -42,6 +43,59 @@ def parse_reads(annotated_fasta, db_type):
...
+
+    start = keys[0]
+    current = start
+    val = coverage[start]
+
+    for k in keys[1:]:
+        if (k-current) == 1 and coverage[k] == val:
+            current = k
+        else:
+            output["%d-%d" % (start, current)] = val
```
Comment (boris-dimitrov) [inline idseq_pipeline/commands/accessionid2seq_functions.py]: I presume these are half-open `[start, current)` intervals.   Worth making sure this is assumed everywhere else.

File: idseq_pipeline/commands/accessionid2seq_functions.py [hunk 2]
```diff
@@ -42,6 +43,59 @@ def parse_reads(annotated_fasta, db_type):
...
+        return output
+    reads = alignment_data['reads']
+    for read in reads:
+        seq = read[1]
+        m8_metrics = read[2]
+        ref_start = int(m8_metrics[-4])
+        ref_end = int(m8_metrics[-3])
+        if ref_start > ref_end: # SWAP
+            (ref_start, ref_end) = (ref_end, ref_start)
+        ref_start -= 1
```
Comment (boris-dimitrov) [inline idseq_pipeline/commands/accessionid2seq_functions.py]: Please explain?

File: idseq_pipeline/commands/accessionid2seq_functions.py [hunk 3]
```diff
@@ -42,6 +43,59 @@ def parse_reads(annotated_fasta, db_type):
...
+        output['total_read_length'] += len(seq)
+        output['total_aligned_length'] += (ref_end - ref_start)
+        output['total_mismatched_length'] += int(m8_metrics[2])
+        output['num_reads'] += 1
+        for bp in range(ref_start, ref_end):
```
Comment (boris-dimitrov) [inline idseq_pipeline/commands/accessionid2seq_functions.py]: Either this range is incorrect or the total_aligned_length computation above is incorrect.

File: idseq_pipeline/commands/accessionid2seq_functions.py (hunk 2 above)
Comment (boris-dimitrov) [reply idseq_pipeline/commands/accessionid2seq_functions.py]: Thanks, I think that would be nice to have in a comment.

**Pull request merged**